include/pocl.h

```diff
@@ -62,6 +62,14 @@ struct _mem_mapping {
   mem_mapping_t *prev, *next;
 };
 
+typedef struct _mem_destructor_callback mem_destructor_callback_t;
+/* represents a memory object destructor callback */
+struct _mem_destructor_callback {
```

Comment:
Check the brace position, opening brace starts from a new line.

http://www.gnu.org/prep/standards/standards.html#Formatting